uportal-war/src/main/webapp/WEB-INF/flows/edit-account/editLocalAccount.jsp

```diff
@@ -62,44 +62,73 @@
                                 <tr>
-                                    <td class="attribute-name"><spring:message code="username"/></td>
-                                    <td><form:input path="username"/></td>
+                                    <td class="attribute-name">
+                                        <label for="${n}userUsername">
+                                            <spring:message code="username" />
+                                        </label>
+                                    </td>
+                                    <td>
+                                        <form:input path="username" id="${n}userUsername" class="form-control"/>
+                                    </td>
                                 </tr>
                             </c:if>
                             <!--  Password and confirm password -->
                             <tr>
-                                <td class="attribute-name"><spring:message code="password"/></td>
-                                <td><form:password path="password"/></td>
+                                <td class="attribute-name">
+                                    <label for="${n}userPassword">
+                                        <spring:message code="password"/>
+                                    </label>
+                                </td>
+                                <td>
+                                    <form:password path="password" id="${n}userPassword" class="form-control"/>
+                                </td>
                             </tr>
                             <tr>
-                                <td class="attribute-name"><spring:message code="confirm.password"/></td>
-                                <td><form:password path="confirmPassword"/></td>
+                                <td class="attribute-name">
+                                    <label for="${n}userConfirmPassword">
+                                        <spring:message code="confirm.password"/>
+                                    </label>
+                                </td>
+                                <td>
+                                    <form:password path="confirmPassword" id="${n}userConfirmPassword" class="form-control"/>
+                                </td>
                             </tr>
-
                         </tbody>
                     </table>
 
                 </div>
             </div>
-        
+
             <!-- Portlet Section -->
             <div class="portlet-section" role="region">
                 <div class="titlebar">
                     <h3 class="title" role="heading"><spring:message code="standard.attributes"/></h3>
                 </div>
                 <div id="${n}standardAttributes" class="content">
-                
+
                     <table class="portlet-table table table-hover">
                         <thead>
```

Comment:
caption is missing, every time you've got thead, you need caption/summary. :wink: